apps/dashboard/src/components/workflow-editor/steps/push/push-tabs.tsx

```diff
@@ -0,0 +1,68 @@
+import { useState } from 'react';
+import { Cross2Icon } from '@radix-ui/react-icons';
+import { RiEdit2Line, RiPencilRuler2Line } from 'react-icons/ri';
+import { useNavigate } from 'react-router-dom';
+
+import { Notification5Fill } from '@/components/icons';
+import { Button } from '@/components/primitives/button';
+import { Separator } from '@/components/primitives/separator';
+import { Tabs, TabsContent, TabsList, TabsTrigger } from '@/components/primitives/tabs';
+import { StepEditorProps } from '@/components/workflow-editor/steps/configure-step-template-form';
+import { PushEditor } from '@/components/workflow-editor/steps/push/push-editor';
+import { WorkflowOriginEnum } from '@novu/shared';
+import { CustomStepControls } from '../controls/custom-step-controls';
+
+const tabsContentClassName = 'h-full w-full overflow-y-auto data-[state=inactive]:hidden';
+
+export const PushTabs = (props: StepEditorProps) => {
```

Comment:
There is a new generic component for all the tabs, see `EmailTabs` or `InAppTabs`

Reply:
Cool, I will update it. Thanks 🙌🏼